README.md

```diff
@@ -0,0 +1,36 @@
+# Avisi Apps Formatter
+
+This is the default formatter for all Avisi Apps projects.
+It is not configurable. It uses [zprint](https://github.com/kkinnear/zprint)
+under the hood.
+
+# Getting started
+
+## Add aliases
+Add `fmt` to your project, add the following to your `deps.edn`
+
+```clojure
+{:aliases {:lint {:override-deps {avisi-apps/fmt {:git/url "git@github.com:avisi-apps/fmt.git"
+                                                  :sha "<git SHA>"}}
+                  :main-opts ["-m" "avisi-apps.fmt.main" "check"]}
+           :fix {:main-opts ["-m" "avisi-apps.fmt.main" "fix"]}}}
+```
+
+## Check formatting
+
+```shell script
+clj -A:lint
+```
+
+## Fix formatting
+
+```shell script
+clj -A:lint:fix
```

Comment:
Kan je hier ook een pad aan meegeven? Zoals we nu doen in het Atlas CRM server project? `clj -A:lint:fix ../`?

Reply:
Nee het format nu altijd alle cljs[cs] files in je project, is er een reden waarom we dat zouden willen?

Reply:
Heb het toch toegevoegd, vooral voor `atlas-crm-server`